templates/kratos.yaml.j2

```diff
@@ -35,10 +50,38 @@ selfservice:
                     - hook: session
         {%- endif %}
     {%- endif %}
-    {%- if oidc_providers %}
+    {%- if oidc_providers or recovery_ui_url or enable_local_idp and login_ui_url %}
     methods:
+        {%- if recovery_ui_url %}
+        code:
+            enabled: True
+        {%- endif %}
+        {%- if not enable_local_idp or not login_ui_url %}
+        password:
+            enabled: False
+        {%- endif %}
+        {%- if enable_local_idp and login_ui_url %}
         password:
-            enabled: false
+            enabled: True
+            config:
+                haveibeenpwned_enabled: False
+        totp:
+            enabled: True
+            config:
+                issuer: Identity Platform
+        {%- if enable_passwordless_login_method %}
+        webauthn:
+            enabled: True
+            config:
+                passwordless: True
```

Comment:
Are there any use cases where we would want to use webauthn as MFA? Or we plan implement this in the login-ui?

Sorry I don't remember exactly what was agreed

Reply:
We decided to only support webauthn as the first auth factor, as it can't be used as MFA simultaneously